01_02_Combat_Rules.txt

```diff
@@ -999,3 +999,128 @@ retrieve a dropped weapon or stand back up if they have fallen.
+entry point to a room.
+
+    In Compound X, the process of clearing a room follows the three steps listed 
+below.
+
+1. Stacking up
+2. Breaching
+3. Clearing
+
+===== Stacking Up ===== 
+
+    Characters can join a stack up line on any time during their turn. Doing so
+immediately ends their turn and the rest of their actions are considered held
+for use during clearing. When a character stacks up, they must indicate exactly
+one entry point that they will use during the breach. If there is any ambiguity,
+they must indicate which room they will storm when the breach occurs. Which room
+you are breaching is usually obvious; it's the room on the other side of the 
+entry point from where your character is stacking up. On the rare occasion that
+one door leads to more than one room, a single room must be designated for 
+clearing. You cannot breach and clear a room that is not attached to the entry
+point that you stacked up on. 
+
+    If your turn starts and you are already stacked up, you may either leave the
+line of the stack up and take your turn normally, or skip your turn to remain in
+the stack up line. If you skip your turn in this manner, you will have the same
+number of actions during clearing as you would if you stacked up as the first
+act of your turn.
+
+    Any number of characters can stack up on a given entry point. The order of
+the characters in the lineup is important. Players can chose any position in the
+stackup order that hasn't yet been taken at the time that they declare their 
+character will stack up. So the first character to stack up could choose to get
+in the middle of the line, and the second character to use their turn to stack
+up could stack up in front if they so chose.  
+
+===== Breaching =====
+
+    Any time a character joins the stack up line, the character in the lead of
+the stack may call "breach!" At that time, regular initiative order is 
+interrupted, and the breach and clear phases occur during the turn of the 
+character who just got in line. After the clearing has resolved, play resumes
+after that character's turn.
+
+    Some weapons, tools or gadgets are commonly used during a breach maneuver.
+Examples include breaching charges, tear gas or flashbangs. If these are used,
+they take effect first in the breach. Keep in mind that a character using such
+a device will need a free hand to do so, and may need to spend one of their 
+clearing actions to re-draw their weapon. As such, devices are usually used by
+the characters in the 3rd or 4th positions.
+
+    Some rooms are defended with booby traps. In the case that a room or entry
+point has such perils, they take effect second. 
+
+    While devices are being resolved, all attackers and defenders roll a reflex
+save to determine the order of combat during the clearing phase. If the 
+defenders were unaware of the impending breach, then they roll reflex with a
+disadvantage and the attackers roll reflex with advantage.
```

Comment:
This seems unnecessarily punishing. I would suggest only defenders-disadvantage for being surprised, and leave attackers-advantage for other circumstances to invoke.